src/js/pie.js

```diff
@@ -21,19 +21,25 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
     // 4. update the pie when the data set changes, including adding or removing slices
 
     fluid.defaults("floe.chartAuthoring.pieChart.pie", {
-        gradeNames: ["floe.chartAuthoring.totalRelaying", "floe.d3ViewComponent",  "autoInit"],
+        gradeNames: ["floe.chartAuthoring.valueBinding","floe.chartAuthoring.totalRelaying", "floe.d3ViewComponent"],
```

Comment:
Oh, really sorry for these (mostly) false alarms. Since I am on my laptop today everything is just a bit too far away to see. Embarrassingly I've just been running linting on the wrong project. But the main thing which brought this up visually was the fact that somehow the option "white": true appears to have gone missing from the core .jshintrc file in infusion. However, with this in, the core framework itself blatantly fails to lint. I'll need to follow up with @jobara and others what the exact history was here, since I'm sure that one of the main reasons for persisting with our incredibly ancient pinned jshint version was precisely to get whitespace linting. Interestingly it is here in the GPII main repo: https://github.com/GPII/universal/blob/master/.jshintrc#L23